src/forms/labels/input-label.js

```diff
@@ -55,34 +57,50 @@ const propTypes = {
   label: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
   name: PropTypes.string.isRequired,
   tooltip: PropTypes.string,
+  required: PropTypes.bool,
+  requiredIndicator: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
   ...togglePropTypes('tooltipShown')
 }
 
 const defaultProps = {
   hint: '',
   label: '',
   tooltip: '',
+  requiredIndicator: '*'
```

Comment:
I'm not sure about this but I think this may retroactively add an indicator to existing lp-component inputs that are flagged as required?